### DISCUSSION TITLE
SID & orgcd 까지 추출했는데 인증오류

### DISCUSSION THREAD

**Discussion by rladbssam317** [changdoc/naver-vaccine-macro#198] [General]
안녕하세요! 좋은 자료 만들어주셔서 감사드립니다! 

#4에 SID와 orgcd까지 추출해내서 reservation 사이트까지 넘어갔는데 네이버인증 다 마쳤는데도 페이지가 넘어가지 않습니다 어떻게 해야돼죠? 

지도에서 매크로를 돌렸을때는 0에서 빨간색으로 깜빠깜빡 거리는데 네이버인증을 먼저 해야된다고 나와있어서 그럼 인증을 안한 상태에서 지도에 매크로 돌리면 안되는거죠? 

![image](https://user-images.githubusercontent.com/88364790/127963138-30661f38-2941-4309-9cd1-3053025bfe58.png)

**Comment by changdoc** [comment]
네 네이버 인증이 안된 상태면 잔여 백신이 탐니되더라도 바로 예약 신청이 안됩니다.

일단 자동 예약 정지 하신 상태에서 본인 인증을 먼저 진행해보시는게 어떨까요?

**Reply by rladbssam317** [reply]
본인인증을 진행해도 그 다음 페이지로 넘어가지 않는데 네이버에서 막은걸까요?ㅠㅠ

**Reply by changdoc** [reply]
본인인증을 했는데도 계속 본인인증을 하라는 페이지가 나온다는 말씀이실까요? 네이버가 인증 자체를 막을 일은 없을거 같습니다.

**Reply by changdoc** [reply]
위에 스크린샷을 보면 매크로가 동작중인 상태로 보이는데요 계속 새로고침이 되는 상태라서 본인 인증 처리가 안 될수도 있겠단 생각이 드네요.

**Reply by rladbssam317** [reply]
제가 모바일에는 네이버인증을 해놓은 상태이고

여기 리스트에 있는 병원에서 하나를 열기 한 후 인증을 하면 저 화면에서 넘어가지 않습니다... 

SID Orgcd로도 해보았는데 둘다 마찬가지로 인증확인으로 넘어가지지 않습니다... 


![image](https://user-images.githubusercontent.com/88364790/127963998-22c79e8d-9899-455b-83c2-e0e7343d0d2c.png)


![image](https://user-images.githubusercontent.com/88364790/127964126-301aa237-a348-433d-901c-555170ece999.png)

**Reply by rladbssam317** [reply]
방금 웨일브라우져에서 주소 복붙해서 네이버인증 진행해보니 웨일에서는 가능한데 크롬에서는 넘어가지질 않네요ㅠㅠ

**Reply by changdoc** [reply]
크롬에서만 인증 진행과정에서 에러가 발생되진 않는지 궁금하네요.
개발자 도구 여셔서 console창에 무슨 에러 메시지들이 있는지 한번 보여주시겠어요?

**Reply by rladbssam317** [reply]
웨일로 일단 해보고있습니다!! 감사합닏ㅏ
